tests/Liquid/StandardFiltersTest.php

```diff
@@ -579,8 +579,10 @@ public function testSort()
 			array('a' => 45, 'b' => 5),
 			array('a' => 40, 'b' => 6),
 			array('a' => 30, 'b' => 48),
+			array('a' => 50),
 		);
 		$expected = array(
+			array('a' => 50),
```

Comment:
Please add another test. We need to be sure it is not a BC break (when existing tests work as before).